packages/client-common/src/__tests__/TestSuite.ts

```diff
@@ -68,6 +68,13 @@ export class TestSuite {
     return client;
   }
 
+  public makeDictionarySearchClient(
+    appIdEnv: string = 'ALGOLIA_APPLICATION_ID_2',
+    apiKeyEnv: string = 'ALGOLIA_ADMIN_KEY_2'
+  ) {
+    return this.makeSearchClient(appIdEnv, apiKeyEnv);
```

Comment:
this method isn't needed, we can use makeSearchClient in the test with these credentials instead :)